inc/classes/preload/class-homepage.php

```diff
@@ -61,7 +61,21 @@ private function get_urls( $url ) {
 
 		$response = wp_remote_get( $url, $args );
 
+		$errors = get_transient( 'rocket_preload_errors' );
```

Comment:
To prevent notices:
```php
$errors           = get_transient( 'rocket_preload_errors' );
$errors           = is_array( $errors ) ? $errors : [];
$errors['errors'] = isset( $errors['errors'] ) && is_array( $errors['errors'] ) ? $errors['errors'] : [];
```